### PR TITLE
Nit-y refactoring and cleanup.

### PR DESCRIPTION
**Pull request by mookerji**
0. Tightening namespaces and other doc nits.
1. Removed dead code.
2. Changes following new hdf5 file format.

WIP

/cc @imh @fnoble

### CODE REVIEW

File: gnss_analysis/agg_run.py [hunk 1]
```diff
@@ -51,11 +52,11 @@ def main():
 
   new_cols = [col for col in reports.keys() if col not in out_df.columns]
   for new_col in new_cols:
-    out_df[new_col] = pd.Series([np.nan]*len(out_df.index), index=out_df.index)
+    out_df[new_col] = pd.Series([np.nan] * len(out_df.index), index=out_df.index)
```
Comment (fnoble) [inline gnss_analysis/agg_run.py]: or better `np.nan * np.empty_like(out_df.index)`?

File: gnss_analysis/data_io.py [hunk 2]
```diff
@@ -111,22 +115,25 @@ def construct_pyobj_eph(eph):
...
+                   eph.prn)
+
 
 def construct_pyobj_sdiff(s):
   """
   Turns sdiff data into the libswiftnav sdiff.
 
   Parameters
-  ----------
+  ----------cal_sat
```
Comment (fnoble) [inline gnss_analysis/data_io.py]: what does `cal_sat` do here?

File: gnss_analysis/data_io.py [hunk 3]
```diff
@@ -197,19 +206,16 @@ def mk_sdiffs_and_abs_pos(ephs, local, remote):
...
   """
-  ephs = ephs.ix[:, [el for el in ephs.major_axis if el != 'payload'],:]
+  ephs = ephs.ix[:, [el for el in ephs.major_axis if el != 'payload'], :]
   obs = sds_with_lock_counts(local, remote)
-  j = obs.transpose(1,0,2).join(
-      local.transpose(1,0,2), rsuffix='_local').join(
-      remote.transpose(1,0,2), rsuffix='_remote').transpose(1,0,2)
-
+  j = obs.transpose(1, 0, 2).join(
+      local.transpose(1, 0, 2), rsuffix='_local').join(
```
Comment (fnoble) [inline gnss_analysis/data_io.py]: s/local/rover?

File: gnss_analysis/data_io.py [hunk 4]
```diff
@@ -197,19 +206,16 @@ def mk_sdiffs_and_abs_pos(ephs, local, remote):
...
+      remote.transpose(1, 0, 2), rsuffix='_remote').transpose(1, 0, 2)
```
Comment (fnoble) [inline gnss_analysis/data_io.py]: s/remote/base?

File: gnss_analysis/data_io.py [hunk 5]
```diff
@@ -318,7 +317,7 @@ def compute_ecef(pseudoranges, dops, sat_poss, sat_vels, t):
     compute it, otherwise an array of NaNs.
   """
   if len(dops) < 4:
-    return [np.nan]*3
+    return [np.nan] * 3
```
Comment (fnoble) [inline gnss_analysis/data_io.py]: would suggest `(np.nan, np.nan, np.nan)` to make it really clear its a 3 vector or nans

File: gnss_analysis/data_io.py [hunk 6]
```diff
@@ -327,16 +326,23 @@ def compute_ecef(pseudoranges, dops, sat_poss, sat_vels, t):
...
   return calc_PVT(nms).pos_ecef
 
+
+# Overwrite will overwrite if sdiffs are not in the table anyway
 def load_sdiffs_and_pos(data_filename,
-                        key_eph='ephemerises', key_local='local', key_remote='remote',
-                        key_local_ecef='local_ecef', key_remote_ecef='remote_ecef',
-                        key_sdiff='sdiffs', overwrite=False):
+                        key_eph='ephemerides',
+                        key_local='rover_obs',
```
Comment (fnoble) [inline gnss_analysis/data_io.py]: s/local/rover
s/remote/base

should probably grep for these across the whole project

File: gnss_analysis/glue.py [hunk 7]
```diff
@@ -198,52 +192,20 @@ def main_tune():
     print 'best found:'
     print settings
 
+
 def main_analyze():
     b = np.array([15, 0, 0])
-    #rough lemnos position:
+    # rough lemnos position:
```
Comment (fnoble) [inline gnss_analysis/glue.py]: Lots of hard coded paths and constants, but I think this file may be deprecated anyway so ask @imh about just removing altogether

Comment (imh) [inline gnss_analysis/glue.py]: yeah, this file is on its way out. I'll take another look and make a new commit to remove it (and some others). They're from the original SITL

File: gnss_analysis/data_io.py (hunk 2 above)
Comment (imh) [inline gnss_analysis/data_io.py]: WTFWTFWTFWTFWTFWTFWTFWTFWTF WHAT DOES IT MEAN??????????

**Comment by mookerji** [comment]
Doing another run through and this will be ready in a bit.

File: gnss_analysis/utils.py [hunk 8]
```diff
@@ -107,5 +108,50 @@ def not_nan(x):
...
     return True
 
+
 def get_non_nans(xs):
-    return xs[xs.apply(not_nan)]
\ No newline at end of file
+  return xs[xs.apply(not_nan)]
+
+
+def truthifyv(phiv):
```
Comment (mookerji) [inline gnss_analysis/utils.py]: TODO (Buro): Double check consuming this from gnss-analysis.

**Comment by mookerji** [comment]
k, this is ready to go.

**Pull request merged**